pkg/virt-api/webhooks/mutating-webhook/mutators/vmi-mutator_test.go

```diff
@@ -194,14 +175,12 @@ var _ = Describe("VirtualMachineInstance Mutator", func() {
 		Expect(vmiSpec.Domain.CPU.Cores).To(Equal(uint32(4)))
 	})
 
-	It("should apply namespace limit ranges on VMI create", func() {
-		_, vmiSpec, _ := getMetaSpecStatusFromAdmit()
-		Expect(vmiSpec.Domain.Resources.Limits.Memory().String()).To(Equal(memoryLimit))
-	})
-
-	It("should apply defaults on VMI create", func() {
+	DescribeTable("should apply defaults on VMI create", func(arch string, cpuModel string) {
 		// no limits wanted on this test, to not copy the limit to requests
-		mutator.NamespaceLimitsInformer, _ = testutils.NewFakeInformerFor(&k8sv1.LimitRange{})
+		defer func() {
+			webhooks.Arch = rt.GOARCH
+		}()
+		webhooks.Arch = arch
```

Comment:
Is this hunk necessary? Looks like it belongs to [`a62049a` (#7007)](https://github.com/kubevirt/kubevirt/pull/7007/commits/a62049a358a25d10d55d5e318820fe2dca612455)

Reply:
Done. Thanks!